doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_name_mappings_interceptor.md

```diff
@@ -92,64 +98,169 @@ Other interceptors may add further headers, such as the `timestamp` header added
+                     "attributeMappings": []
+                  }
+               ]
+            },
+            {
+               "originalServicePath": "/other",
+               "entityMappings": [
+                  {
+                     "originalEntityId": "room.(\\d*)",
+                     "originalEntityType": "room",
+                     "newEntityId": "all_other",
+                     "newEntityType": "room",
+                     "attributeMappings": []
+                  }
+               ]
+            }
+         ]
+      }
+   ]
+}
 ```
 
 Now, let's assume the following not-intercepted event regarding a received notification (the code below is an <i>object representation</i>, not any real data format):
 
-    flume-event={
-        headers={
-	         fiware-service=hotel,
-	         fiware-servicepath=/allrooms,/allrooms,
-	         transaction-id=1234567890-0000-1234567890,
-	         correlation-id=1234567890-0000-1234567890
-        },
-        body=[
-           {
-	            entityId=Room.12,
-	            entityType=Room.left,
-	            attributes=[
-	                ...
-	            ]
-	        },
-	        {
-	            entityId=Room2,
-	            entityType=Room,
-	            attributes=[
-	                ...
-	            ]
-	        }
-	    ]
-    }
-
-As can be seen, two entities (`Room1` and `Room2`) of the same type (`Room`) within the same FIWARE service (`hotel`) and service paths (`/allrooms` for both of them) are notified. Once intercepted, this is how the event should look like (the code below is an <i>object representation</i>, not any real data format):
-
-    flume-event={
-        headers={
-	         fiware-service=hotel,
-	         fiware-servicepath=/allrooms,/allrooms,
-	         transaction-id=1234567890-0000-1234567890,
-	         correlation-id=1234567890-0000-1234567890,
-	         timestamp=1234567890,
-	         mapped-fiware-service=
-	         mapped-fiware-service-path=
-        },
-        body=[
-           {
-	            entityId=Room.12,
-	            entityType=Room.left,
-	            attributes=[
-	                ...
-	            ]
-	        },
-	        {
-	            entityId=Room2,
-	            entityType=Room,
-	            attributes=[
-	                ...
-	            ]
-	        }
-	    ]
-    }
+```
+notification={
+   headers={
+	   fiware-service=hotel1,
+	   fiware-servicepath=/other,/suites,
+	   correlation-id=1234567890-0000-1234567890
+   },
+   body={
+      {
+	      entityId=suite.12,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   },
+	   {
+	      entityId=other.9,
+	      entityType=room,
+	      attributes=[
+	         ...
+	      ]
+	   }
+	}
+}
+```
+
+As can be seen, two entities (`suite.12` and `other.9`) of the same type (`room`) within the same FIWARE service (`hotel`) but different service paths (`/suites` and `/other`) are notified. `NGSIRestHandler` will crate two `NGSIEvent`'s:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af